std/assembly/array.ts

```diff
@@ -563,7 +563,7 @@ export class Array<T> {
 
   // RT integration
 
-  @unsafe private __visit_impl(cookie: u32): void {
+  @unsafe private __visit(cookie: u32): void {
```

Comment:
Perfection is achieved not when there is nothing more to add, but when there is nothing left to take away.